.github/workflows/release.yml

```diff
@@ -0,0 +1,38 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    name: Release
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      # Specify a different token to allow pushing changes to main later
+        with:
+          token: ${{ secrets.ADMIN_GITHUB_TOKEN }}
+      - name: Set Git user
+        run: |
+          git config --global user.email "action@github.com"
+          git config --global user.name "Github Action"
+      - name: Publish Release
+        id: release
+        uses: release-drafter/release-drafter@v5
+        with:
+          publish: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.ADMIN_GITHUB_TOKEN }}
```

Comment:
Also, just so that it is clear, this only works because we have added an organization-level secret to our SU-SWS org for the value. New secrets will have to be added per repo.